packages/blade/src/components/Checkbox/CheckboxIcon/styles.css

```diff
@@ -0,0 +1,23 @@
+@keyframes fadeIn {
```

Comment:
Couldn't figure out how to get styled component's Keyframe to work with the native CSS animations thus added styles.css file. 

Will this cause any issues? If so then I'll relook it.

Reply:
check this section in rfc you can do that as you write regular CSS - https://github.com/razorpay/blade/blob/master/rfcs/2022-03-22-motion-rfc.md#web-react

Reply:
Will check.

Reply:
Fixed with keyframe from styled components.